awscli/clidriver.py

```diff
@@ -455,6 +473,12 @@ def lineage_names(self):
         # Represents the lineage of a command in terms of command ``name``
         return [cmd.name for cmd in self.lineage]
 
+    @property
+    def subcommand_table(self):
+        # There's no subcommands for an operation so we return an
```

Comment:
Technically a subcommand in the sense that you can add `help` to the end of any command to get help, but not in terms of what's considered the `subcommand_table` in the CLI.  None of the built in subcommand tables nor the subcommand tables used for custom commands add `help` as a subcommand so it would be inconsistent to add `help` here.